src/languages/es-ES.ts

```diff
@@ -3774,7 +3774,7 @@ export default class extends Language {
 			`→ ${inlineCodeblock('Type  ::')} **Twemoji**`,
 			`→ ${inlineCodeblock('ID    ::')} **${id}**`
 		].join('\n'),
-		COMMAND_EMOJI_INVALID: emoji => `'${emoji}' no es un emoji válido.`,
+		COMMAND_EMOJI_INVALID: `The argument you provided is not a valid emoji.`,
```

Comment:
Did you mean to put the English translation here?

Reply:
Fixed in 8e222c81